app/presenters/api/application_presenter.rb

```diff
@@ -0,0 +1,33 @@
+module API
+  class ApplicationPresenter < Base
+    VERSION = '1.1'
+
+    VERSIONS = {
+      '1.2' => ['MoreApplicationData'],
```

Comment:
Would it be possible and less error prone to make these arrays of modules instead of strings? Not sure if load order would prevent this.

Reply:
Addressed in the PR implementing versioning - https://github.com/DFE-Digital/apply-for-teacher-training/pull/6170